<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashierWindow</class>
 <widget class="QDialog" name="CashierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1051</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="showtimes_tab">
       <attribute name="title">
        <string>Showtimes</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>1031</width>
          <height>681</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QTableWidget" name="showtimes_table">
           <column>
            <property name="text">
             <string>Movie</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Hall</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date and time</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="reservations_tab">
       <attribute name="title">
        <string>Reservations</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>1031</width>
          <height>681</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QTableWidget" name="reservations_table">
           <column>
            <property name="text">
             <string>Surname</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Email</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Seat</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="process_btn">
           <property name="text">
            <string>Process</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tickets_tab">
       <attribute name="title">
        <string>Tickets</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1031</width>
          <height>681</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tickets_table">
           <column>
            <property name="text">
             <string>Surname</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Seat</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Price</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="halls_tab">
       <attribute name="title">
        <string>Seatings</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1031</width>
          <height>681</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QComboBox" name="show_combo"/>
           </item>
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>400</height>
              </size>
             </property>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <column>
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <color>
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <item row="0" column="1">
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
              <property name="foreground">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="2">
              <property name="text">
               <string/>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="background">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="3">
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="4">
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="5">
              <property name="text">
               <string>SCREEN</string>
              </property>
              <property name="background">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="6">
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="7">
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="8">
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="0" column="9">
              <property name="text">
               <string/>
              </property>
              <property name="background">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>255</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="4" column="1">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="4" column="2">
              <property name="text">
               <string>Seat 0</string>
              </property>
             </item>
             <item row="4" column="3">
              <property name="text">
               <string>Seat 1</string>
              </property>
             </item>
             <item row="4" column="4">
              <property name="text">
               <string>Seat 2</string>
              </property>
             </item>
             <item row="4" column="5">
              <property name="text">
               <string>Seat 3</string>
              </property>
             </item>
             <item row="4" column="6">
              <property name="text">
               <string>Seat 4</string>
              </property>
             </item>
             <item row="4" column="7">
              <property name="text">
               <string>Seat 5</string>
              </property>
             </item>
             <item row="4" column="8">
              <property name="text">
               <string>Seat 6</string>
              </property>
             </item>
             <item row="4" column="9">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="5" column="1">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="5" column="2">
              <property name="text">
               <string>Seat 7</string>
              </property>
             </item>
             <item row="5" column="3">
              <property name="text">
               <string>Seat 8</string>
              </property>
             </item>
             <item row="5" column="4">
              <property name="text">
               <string>Seat 9</string>
              </property>
             </item>
             <item row="5" column="5">
              <property name="text">
               <string>Seat 10</string>
              </property>
             </item>
             <item row="5" column="6">
              <property name="text">
               <string>Seat 11</string>
              </property>
             </item>
             <item row="5" column="7">
              <property name="text">
               <string>Seat 12</string>
              </property>
             </item>
             <item row="5" column="8">
              <property name="text">
               <string>Seat 13</string>
              </property>
             </item>
             <item row="5" column="9">
              <property name="text">
               <string/>
              </property>
             </item>
             <item row="6" column="2">
              <property name="text">
               <string>Seat 14</string>
              </property>
             </item>
             <item row="6" column="3">
              <property name="text">
               <string>Seat 15</string>
              </property>
             </item>
             <item row="6" column="4">
              <property name="text">
               <string>Seat 16</string>
              </property>
             </item>
             <item row="6" column="5">
              <property name="text">
               <string>Seat 17</string>
              </property>
             </item>
             <item row="6" column="6">
              <property name="text">
               <string>Seat 18</string>
              </property>
             </item>
             <item row="6" column="7">
              <property name="text">
               <string>Seat 19</string>
              </property>
             </item>
             <item row="6" column="8">
              <property name="text">
               <string>Seat 20</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QTextBrowser" name="taken_seats_text">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="add_ticket_tab">
       <attribute name="title">
        <string>Add ticket</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>1021</width>
          <height>671</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="1" colspan="3">
          <widget class="QTextBrowser" name="price_table">
           <property name="maximumSize">
            <size>
             <width>5000</width>
             <height>250</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Showtime</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="noadults_input"/>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="nochildren_input"/>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="surname_input"/>
         </item>
         <item row="0" column="4">
          <widget class="QLineEdit" name="seats_input"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Seats</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Number of adults</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="name_input"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLineEdit" name="nostudents_input"/>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="showtime_combobox"/>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Number of children</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLineEdit" name="email_input"/>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Number of students</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="QPushButton" name="add_ticket">
           <property name="text">
            <string>Print bill and add ticket</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Surname</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
